src/projects/ResponsivePhotographyWebsite/index.js

```diff
@@ -0,0 +1,130 @@
+import styles from "./styles.module.css"
+import {useEffect} from "react";
+
+const ResponsivePhotographyWebsite = () => {
+    useEffect(() => {
+        document.title = 'Photography';
```

Comment:
You were going to change an icon too? Didn't manage to find a solution?

Reply:
not found fast and eazy solution, without 3rd party libraries, so decided to decline this idea